source/common/http/conn_manager_impl.cc

```diff
@@ -909,6 +909,40 @@ bool ConnectionManagerImpl::ActiveStream::validateHeaders() {
   return true;
 }
 
+bool ConnectionManagerImpl::ActiveStream::validateTrailers() {
+  if (header_validator_) {
```

Comment:
if (!header_validator_) {
  return true
}
and unindent all below?

Reply:
Done